projects/igniteui-angular/src/lib/grids/row.component.ts

```diff
@@ -303,7 +302,7 @@ export class IgxRowComponent<T extends IgxGridBaseComponent & IGridDataBindable>
      * This method emits `onEditDone` event.
      *
      * ```typescript
-     * // update the second selected row's value
+    *  // update the second selected row's value
```

Comment:
That didn't quite work.
```suggestion
     * // update the second selected row's value
```